core/src/main/java/io/bisq/core/btc/wallet/WalletsSetup.java

```diff
@@ -300,35 +300,41 @@ private void configPeerNodesForLocalHostBitcoinNode() {
 
     private void configPeerNodes(Socks5Proxy socks5Proxy) {
         List<BitcoinNodes.BtcNode> btcNodeList = new ArrayList<>();
-        walletConfig.setMinBroadcastConnections((int) Math.floor(DEFAULT_CONNECTIONS * 0.8));
         switch (BitcoinNodes.BitcoinNodesOption.values()[preferences.getBitcoinNodesOptionOrdinal()]) {
             case CUSTOM:
-                String bitcoinNodesString = preferences.getBitcoinNodes();
-                if (bitcoinNodesString != null) {
+                final String bitcoinNodesString = preferences.getBitcoinNodes();
+                if (bitcoinNodesString != null && !bitcoinNodesString.isEmpty()) {
```

Comment:
As far as I can tell, it is impossible for `bitcoinNodesString` to be empty. If the user provides `--btcNodes=""` at the command line, the empty string value gets rejected at command line option parsing time.

What the user *can* do is provide `--btcNodes=","`, which is effectively an empty list of nodes, but this logic doesn't capture it, because the raw `bitcoinNodesString` is literally `,`, and this is not an empty string. This leads to a bug where the list of custom nodes in the app is literally `,`, but Bisq still ends up connecting to the public network. I didn't track down exactly why that happens, because clearly this guard logic should be reworked anyway.

Reply:
I preferred to not relay on the default value from the program arguments as this class does not know really about that.
The .filter(e -> !e.isEmpty()) should handle the case if we get a list of empty strings.
I improved the ocde via commit 9ac49bd91a2789e0db56d91a64b567a18bf9e951

Reply:
Added a test as well.